testProjectNative/build.gradle

```diff
@@ -0,0 +1,36 @@
+// A C++ project that demonstrates most of the configuration features of this
+// plugin.
+
+apply plugin: 'cpp'
+apply plugin: 'com.google.protobuf'
+
+model {
+    components {
+        main(NativeLibrarySpec)
+        test(NativeExecutableSpec) {
+            binaries.all {
+                lib library: 'main', linkage: 'static'
+            }
+        }
+    }
+
+    protobuf {
+        protoc {
+            artifact = 'com.google.protobuf:protoc:2.6.1'
```

Comment:
Since the user has to specify `PROTOBUF_HOME` anyway, having to specify a separate `protoc` artifact is redundant and adds chance of using incompatible versions of `protoc` and the protobuf library. I would instead infer the `protoc` path from `PROTOBUF_HOME` by default.